hwdb.d/70-mouse.hwdb

```diff
@@ -49,6 +50,15 @@
 #   MOUSE_WHEEL_CLICK_COUNT_HORIZONTAL
 #
 #########################################
+#         ID_INPUT_3D_MOUSE             #
+#########################################
+#
+# Specified if the device is a 3D mouse[1]. This gives access to the
+# hidraw device node to applications that consume its events directly.
```

Comment:
Please add a comment like for ID_INPUT_TRACKBALL (lines 55 in the old version).

Reply:
This is all that's needed, the comments about `ID_INPUT_MOUSE` don't apply to this type of device.